plugins/GranularPitchShifter/GranularPitchShifterControlDialog.cpp

```diff
@@ -28,7 +28,7 @@
 #include "LcdFloatSpinBox.h"
 #include "Knob.h"
 #include "GuiApplication.h"
-#include "gui_templates.h"
```

Comment:
self note: I don't think granular pitch shifter uses the function, so i'll try removing once the build succeeds.